samples/IISSample/web.config

```diff
@@ -1,9 +1,23 @@
-﻿<?xml version="1.0"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+
+  <!--
+    Configure your application settings in appsettings.json. Learn more at http://go.microsoft.com/fwlink/?LinkId=786380
+  -->
+
   <system.webServer>
```

Comment:
Include this comment from the test web.configs:
`<!-- This repository cannot use the iis publish tool because there's a bug in the installer if the tool package is a project reference -->`